packages/schedule/src/ScheduleContainer.spec.tsx

```diff
@@ -53,17 +55,16 @@ describe('ScheduleContainer', () => {
   it('updates elapsed render prop on each raf call', () => {
     const now = Date.now();
     const { getByTestId } = render(<BasicExample />);
-    let spy;
 
     act(() => {
-      spy = jest.spyOn(Date, 'now').mockImplementationOnce(() => now - 1000);
+      const spy = jest.spyOn(Date, 'now').mockImplementationOnce(() => now - 1000);
+
       jest.runOnlyPendingTimers();
       jest.advanceTimersByTime(1000);
+      spy.mockRestore();
```

Comment:
Is there a reason that the `mockRestore` had to be moved to the `act` call?

Reply:
I was running into a error message "variable 'spy' is used before being assigned" and figured I'd move it into the `act` call. But I'm not currently familiar with the internals of the `act` call and I probably shouldn't do this. Thanks for pointing it out.

![Screen Shot 2019-10-28 at 3 43 28 PM](https://user-images.githubusercontent.com/1811365/67723651-ea6f8380-f999-11e9-90d4-8f1b098fa741.png)

I'll revert this back and use a `any` type because this does not cause an error.

```ts
 it('updates elapsed render prop on each raf call', () => {
    const now = Date.now();
    const { getByTestId } = render(<BasicExample />);
    let spy: any;

    act(() => {
      spy = jest.spyOn(Date, 'now').mockImplementationOnce(() => now - 1000);

      jest.runOnlyPendingTimers();
      jest.advanceTimersByTime(1000);
    });

    spy.mockRestore();

    expect(getByTestId('schedule').textContent).not.toBe('0');
  });
```

@austin94 would you have a suggested workaround? I'll keep playing around with this.

Reply:
This makes sense! Just wasn't sure which rule changed from `js -> ts`. Feel free to leave this.